src/Components/TableComponents/NewMatchesTableRow.js

```diff
@@ -57,6 +57,15 @@ export const NewMatchesTableRow = ({
             },
           },
         });
+
+        if (
```

Comment:
This is just a nitpick, looking at it, it might be difficult to understand. 

We can add a note explaining that 
`If dropdown is changed from not reviewed to anything, decrease counter/call fetch, and if we go from a random value to not review, increment/call the fetch`